test/resources/demo-users/user/ns2/ec-integration-test.yaml

```diff
@@ -17,9 +17,9 @@ spec:
   resolverRef:
     params:
       - name: url
-        value: 'https://github.com/gbenhaim/build-definitions'
+        value: 'https://github.com/konflux-ci/build-definitions'
       - name: revision
-        value: 997f4b966aae7d65e958e63ff5b1dbdd58e4dbc6
+        value: main
```

Comment:
[A PR](https://github.com/konflux-ci/konflux-ci/pull/195/files#diff-f96d8721cb200706c5e4a118974dcc4fd174144c71762da0a5360268f988a649) was merged lately, changing the revision on multiple files from using branch name to using commit sha.

Check what's the latest commit on the main branch of https://github.com/konflux-ci/build-definitions and use its sha for `value` instead of `main`.

Reply:
@yftacherzog PTAl now. Updated it to the latest commit